package.json

```diff
@@ -91,6 +91,39 @@
     "webpack-dev-server": "^1.14.1",
     "yuglify": "~0.1.4"
   },
+  "jest": {
+    "verbose": true,
+    "rootDir": "/opt/dev/troposphere",
```

Comment:
in the Github branch, there is no `/opt/dev/` and this actually make the test fails (however CI doesn't recognize it since CI thinks no test script existed actually. 

Suggest to change it to an absolute path.